sdk/storage/storage-queue/src/StorageClient.ts

```diff
@@ -59,10 +59,6 @@ export abstract class StorageClient {
     this.url = url;
     this.accountName = getAccountNameFromUrl(url);
     this.pipeline = pipeline;
-    this.storageClientContext = new StorageClientContext(url, pipeline.toServiceClientOptions());
-
-    // Override protocol layer's default content-type
-    const storageClientContext = this.storageClientContext as any;
-    storageClientContext.requestContentType = undefined;
```

Comment:
Defined `getStorageClientContext` in utils.common.ts file so that it can be reused for messagesContext and messageIdContext